src/server/page.ts

```diff
@@ -428,6 +438,14 @@ export class Page extends EventEmitter {
     this._video = video;
     this.emit(Page.Events.VideoStarted, video);
   }
+
+  frameNavigated(frame: frames.Frame) {
+    this.emit(Page.Events.FrameNavigated, frame);
+    const url = frame.url();
+    if (url === 'about:blank' || url.startsWith('chrome:'))
```

Comment:
Do we want origins for data: urls, file: urls, other about: urls? Perhaps `!url.startsWith('http')` ?